src/components/ContentHighlights/ContentHighlightsDashboard.jsx

```diff
@@ -1,17 +1,47 @@
-import React, { useContext } from 'react';
+import React, { useContext, useEffect, useState } from 'react';
 import PropTypes from 'prop-types';
-import { Container } from '@edx/paragon';
+import { useHistory } from 'react-router-dom';
+import { Container, Toast } from '@edx/paragon';
+
 import ZeroStateHighlights from './ZeroState';
 import CurrentContentHighlights from './CurrentContentHighlights';
 import ContentHighlightHelmet from './ContentHighlightHelmet';
 import { EnterpriseAppContext } from '../EnterpriseApp/EnterpriseAppContextProvider';
 
-const ContentHighlightsDashboardBase = ({ children }) => (
-  <Container fluid className="my-5">
-    <ContentHighlightHelmet title="Highlights" />
-    {children}
-  </Container>
-);
+const ContentHighlightsDashboardBase = ({ children }) => {
+  const history = useHistory();
+  const { location } = history;
+  const { state: locationState } = location;
+
+  const [hasDeletedHighlightSetToast, setHasDeletedHighlightSetToast] = useState(false);
+
+  // TODO: the below `useEffect` needs test coverage. deferred until there is a reducer
+  // for the `ContentHighlights` module, where `DeleteHighlightSet` can dispatch an action
+  // to trigger the `Toast`, rather than relying on history's location state.
+  /* istanbul ignore next */
+  useEffect(() => {
+    if (!locationState?.deletedHighlightSet) {
+      return;
+    }
+    setHasDeletedHighlightSetToast(true);
+    const newState = { ...locationState };
+    delete newState.deletedHighlightSet;
+    history.replace({ ...location, state: newState });
+  }, [history, location, locationState]);
+
+  return (
+    <Container fluid className="my-5">
+      <ContentHighlightHelmet title="Highlights" />
+      {children}
+      <Toast
+        onClose={() => setHasDeletedHighlightSetToast(false)}
+        show={hasDeletedHighlightSetToast}
+      >
+        Highlight collection deleted.
```

Comment:
[inform]: currently a reducer has been implemented within hu/ent-6501 branch that has been created in the mean time so that the lastDeletedHighlight will be tracked to populate text with the name of the deleted highlight.

Reply:
yep! we'll need to adjust this approach once introducing that reducer.